test/views/multiyear/transfer/TransferTest.scala

```diff
@@ -438,7 +442,7 @@ class TransferTest extends BaseTest with NinoGenerator {
       val todayPlusOneDay = LocalDate.now().plusDays(1)
       val pattern = "d MM YYYY"
       val todayPlusOneDayFormatted = todayPlusOneDay.format(DateTimeFormatter.ofPattern(pattern))
-      val localDate = LocalDate.now().plusYears(1)
+      val localDate = LocalDate.now().plusYears(2)
```

Comment:
This feels like we're testing a change of behaviour here? Is this test not working at 1 year

Reply:
Test code not removed in error, corrected.